### DISCUSSION TITLE
How does earthaccess handle 'point' data?

### DISCUSSION THREAD

**Discussion by BriannaLind** [earthaccess-dev/earthaccess#417] [Q&A]
while looking at outputs from [NASA-Earth-Data](https://github.com/opengeos/NASA-Earth-Data) I noticed that bounding box information was missing for [this dataset](https://daac.ornl.gov/cgi-bin/dsviewer.pl?ds_id=1) from ORNL @rupesh2 ..... that got me wondering, how does earthaccess handle point data and does the lack of a bounding box affect querability negatively?@giswqs

**Comment by rupesh2** [comment]
@BriannaLind you found the very first dataset that was published by ORNL DAAC :). Spatial subsetting seems to work. See an example below, which will return 7 granules: 

```python
import earthaccess
auth = earthaccess.login(strategy="netrc")
granules = earthaccess.search_data(
    count=-1,
    doi='10.3334/ORNLDAAC/1',
    bounding_box=(-97, 38, -95, 40) ,
    temporal=('1985-01-01','1985-06-01')
)
```
